### ISSUE TITLE
Unable to 'list all my secrets'

### ISSUE THREAD

**Issue by mxm222** [gopasspw/gopass#1617]
### Summary 
I cannot _see_ secrets I've inserted. Running `gopass` like described in [the features documentation](https://github.com/gopasspw/gopass/blob/master/docs/features.md#listing-existing-secrets) just brings up some kind of interactive mode:
```
$ gopass
gopass>
```
Also `gopass show` behaves differently:
```
$ gopass show

Error: Usage: gopass show [name]
```
I've seen `gopass ls`, but it only lists directories, e.g. if I have secrets `foo/bar/a` and `foo/bar/b` the output is
```
$ gopass ls
gopass
└── foo
    └── bar
``` 

### Steps To Reproduce

* Install gopass on Fedora 32 following [the setup guide](https://github.com/gopasspw/gopass/blob/master/docs/setup.md#redhat--centos)
* Set up gopass with `gopass init --path /home/username/.password-store` (replacing `username` with the user's name)
* Insert the secrets from the summary
* Try to list the secrets using `gopass` or `gopass show` or `gopass ls`

### Expected behavior
As described in [the features documentation](https://github.com/gopasspw/gopass/blob/master/docs/features.md#listing-existing-secrets), `gopass` should list the secrets in a tree-view.

### Environment

- OS: Fedora 32
- OS version: 5.8.4-200.fc32.x86_64
- gopass Version: 1.10.1
- Installation method: `dnf`

**Comment by dominikschulz** [comment]
Sorry, I can't reproduce this with `1.10.1`.

You might want to try `GOPASS_DEBUG_LOG=/tmp/logfile` to get a debug log.

The documentation still needs to be updated to cover the interactive mode.

**Comment by mxm222** [comment]
Thanks for the response. I can't see anything suspicious in the log file. I'm attaching it (time stamp `13:23` is the log for `gopass show` and `13:28` for `gopass ls`), maybe there is something obvious I'm overlooking here. I appreciate the help.
[gopass.log](https://github.com/gopasspw/gopass/files/5358913/gopass.log)

**Comment by AnomalRoil** [comment]
This is strange since your logfile seems to indicate that Gopass found 2 secrets in your store.
It should display: 
```
gopass         
└── money/
    └── ing/
        └── cards/
            ├── giro
            └── visa
```

Your config seems sane, I don't see what could cause an issue.

What exact commands are you running and what is the output exactly?

**Comment by mxm222** [comment]
That's what I'd expected to be displayed. The commands and steps to reproduce are described in the opening post. The actual output of `gopass ls` is:
```
$ gopass ls
gopass
└── money
    └── ing
        └── cards
```
But, as you already pointed out, the secrets are there:
```
$ tree ~/.password-store/
.password-store/
└── money
    └── ing
        └── cards
            ├── giro.gpg
            └── visa.gpg

3 directories, 2 files
```

**Comment by AnomalRoil** [comment]
I think this is related to #1628 and solved by #1651 

Closing since it should be solved.
